### PR TITLE
[Fix](from_unixtime) Keep consistent with MySQL & bug fix

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
1. Bug fix: implicit convert from int32 -> int64 makes negative time stamp valid, so change signature to int64
2. Consistent: keep consistent with mysql.

Not merge to 2.0, because it changed function signature.

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/rewrite/FEFunctions.java [hunk 1]
```diff
@@ -231,21 +231,21 @@ public static IntLiteral unixTimestamp2(LiteralExpr arg) throws AnalysisExceptio
         return new IntLiteral(unixTime, Type.INT);
     }
 
-    @FEFunction(name = "from_unixtime", argTypes = { "INT" }, returnType = "VARCHAR")
+    @FEFunction(name = "from_unixtime", argTypes = { "BIGINT" }, returnType = "VARCHAR")
     public static StringLiteral fromUnixTime(LiteralExpr unixTime) throws AnalysisException {
         // if unixTime < 0, we should return null, throw a exception and let BE process
-        if (unixTime.getLongValue() < 0 || unixTime.getLongValue() >= Integer.MAX_VALUE) {
+        if (unixTime.getLongValue() < 0 || unixTime.getLongValue() > 32536771199L) {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/rewrite/FEFunctions.java:237]: pls add some comment about the magic number 32536771199L. And do same in nereids

Comment (zhiqiang-hhhh) [reply fe/fe-core/src/main/java/org/apache/doris/rewrite/FEFunctions.java:238]: done

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request merged**